corehq/apps/users/tests/test_migrate_roles_to_sql.py

```diff
@@ -98,6 +97,49 @@ def test_sync_role_sql_to_couch(self):
         self.assertEqual(couch_role.permissions, sql_role.permissions)
         self.assertEqual(couch_role.assignable_by, sql_role.assignable_by)
 
+    def test_sync_role_sql_to_couch_set_permissions(self):
+        sql_role = SQLUserRole(
+            domain=self.domain,
+            name="test_sql_to_couch",
+            default_landing_page=ALL_LANDING_PAGES[0].id,
+            is_non_admin_editable=False,
+            upstream_id=self.app_editor_sql.couch_id
+        )
+        sql_role.save()
+        couch_roles = UserRole.by_domain(self.domain)
+        self.assertEqual(len(couch_roles), 2)
+        couch_role = [r for r in couch_roles if r.name == sql_role.name][0]
```

Comment:
This is hard to understand without extra context. Why are there 2 couch roles? Is that part of the test setup? Why is the first one special? Would it be possible to change it identify roles by name rather than magic numbers? Something like
```suggestion
        couch_roles = {r.name: r for r in UserRole.by_domain(self.domain)}
        self.assertEqual(couch_roles.keys(), {"test_sql_to_couch", "something_else"})  # <-- the second one is certainly incorrect
        couch_role = couch_roles["test_sql_to_couch"]
```

Reply:
> This is hard to understand without extra context. Why are there 2 couch roles? Is that part of the test setup? Why is the first one special? Would it be possible to change it identify roles by name rather than magic numbers? Something like

There is one role created during setup. I've updated the tests to fetch the role under test directly instead of via the view: ed09c94e078